docs/source/using/addons.adoc

```diff
@@ -307,6 +307,22 @@ $ minishift addons uninstall admin-user
 Add-on 'admin-user' uninstalled
 ----
 
+[[available-addons]]
+== Available Add-ons
```

Comment:
There is some lead-in missing. We have not even introduced the minishift-addons repository and suddenly we list add-ons available from this repo? How does the user get them? How do you install them? We will need to duplicate some of the information from the minishift-addons README here. I also argue this section should come last.